src/lib/cli/rules/templates/external-rule/files/LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We don't need this file. We used to have it because it was a repo in `sonarwhal` but it's not needed now.